src/test/operate.test.js

```diff
@@ -0,0 +1,21 @@
+import operate from 'logic/operate';
+
+describe('Testing Operation', () => {
+  test('operate', () => {
+    const data = operate('2', '4', '+');
+    expect(data).toBe('6');
+  });
+
+  test('operate', () => {
+    const data = operate('8', '4', '-');
+    expect(data).toBe('4');
+  });
+  test('operate', () => {
+    const data = operate('8', '4', 'x');
+    expect(data).toBe('32');
+  });
+  test('operate', () => {
+    const data = operate('8', '4', '÷');
+    expect(data).toBe('2');
+  });
+});
```

Comment:
- [ ] Good job adding tests for `operate.js` logic, however, we have some tests that have not been covered. Kindly ensure you include tests to check:

- That a number cannot be divided by o
![image](https://github.com/tumainimaganiko/math-magicians/assets/81932478/ad44e3a4-7d3c-4a7c-ba70-d9f33732afd8)

- That the correct value is returned for the `%` modulus operation
e.g
`5%3 = 2`

- That you cannot find the `%` modulo of `0` 
![image](https://github.com/tumainimaganiko/math-magicians/assets/81932478/c3e1d852-199e-4ae9-a029-e3520f73b7c4)

- That unknown operations cannot be performed
![image](https://github.com/tumainimaganiko/math-magicians/assets/81932478/f2f7dbfc-5c0d-4c36-b313-4138a832a20d)